### ISSUE TITLE
b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#147]
**Description:**
Same as 017 (Issue: https://github.com/NCAR/amwg_dev/issues/145) but with
```
 clubb_c8               =  5.2 
```
The goal being to bring increase the cloud and bring the RESTOM closer to zero. 

This similar to #146 but using another set of parameters to retune RESTOM. 

This is based on Adam's comment: 
_If you recall all the way back Jiang Zhu's talk on the Eocene simulations in April ([here](https://drive.google.com/file/d/11bCewR7HdpQeLPlUGPY512CC314H_wEz/view)), C8 is an alternative way to impact the skewness. clubb_gamma hits it really hard because it controls a PDF parameter - the width of the w' distribution. But C8 does it through a more natural way, in that it is the damping coefficient on the prognostic w'^3 equation. Increasing C8 will reduce skewness, which will increase cloud thickness._

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019

On github:
https://github.com/NCAR/amwg_dev/tree/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cesm3_cam6_3_058_MOM_e

On github:
hash: https://github.com/NCAR/amwg_dev/commit/8f70c084f85987a02a88df6570c8499a62fbebd9

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/B1850MOM/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019/atm/](https://webext.cgd.ucar.edu/B1850MOM/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019/atm/)

**Contacts:**
@cecilehannay, @gustavo-marques, @JulioTBacmeister, @adamrher

**Comment by JulioTBacmeister** [comment]
RESTOM has been reduced too much in 018 and 019.  Looks like choosing clubb_gamma, clubb_c14, clubb_c8 halfway bewteen the values in 017 and (018, 019) might be warranted.  I'd stop 018 and 019 and try again.  Not sure whether to reuse names ... maybe call new runs 018b and 019b

![image](https://user-images.githubusercontent.com/9221710/181916822-5852f58d-d3ed-4981-b1a1-e903c5dd2b65.png)

**Comment by adamrher** [comment]
How about for 18b we increase clubb_gamma by a little less than half = 0.270 -> 0.280, and than maybe revert c14 back to the 017 run, = 1.6->2.2?

And for 19b, lets reduce c8 by a little more than half way = 5.2->4.6 (since this run looks to be the colder of the two)

**Comment by JulioTBacmeister** [comment]
Sounds reasonable.

**Comment by cecilehannay** [comment]
Sounds like a plan

**Comment by cecilehannay** [comment]
> For 18b let's go with clubb_gamma->0.280 and c14-> 2.2
Indeed, looking at the numbers, I think that going back to c14-> 2.2 might be too much. I would have gone half way. But it is hard to tell as the runs are so short. 
That said, I would rather go a bit too warm than a too too cold.

**Comment by adamrher** [comment]
yeah, I think that's partly why I pressed to revert c14=2.2 --because as you say we'd rather be a bit warm than cold. Also, I'm a little weary of small c14 values since it is a dissipation coefficient for one of the prognostic eqns. Due to the various clubb issues we've uncovered recently, I'm beginning to fear clubb is on the cusp of stability. And so I get nervous when we lower the dissipation coefficients. 

Likewise, Im going to be advocating for increasing c8 permanently, based on Jiang's work, but let's see how the sim's pan out first.
